assets/js/app.js

```diff
@@ -90,7 +90,7 @@ function homeSlider() {
     var viewportWidth = window.innerWidth;
     var carousel = document.getElementById('carousel');
     if (!carousel) return;
-    
+    carousel.style.width =  window.innerWidth  + 'px';
```

Comment:
I think it slows down the automatic scrolling too much. If you remove this, does it break the slider?

Reply:
Yes indeed it does slow down the carousel speed, but without it the carousel break. There's obviously a better solution for that! If you can think of something, let me know!